hooks/proprietary/PBVE.js

```diff
@@ -34,85 +39,229 @@ $PBVE,x,x,xxxx,xxxx,xx,xx,xx,xx,xxxx,xxxx,xx,xxxx,xxxx,xx*xx
 
 0: Product code: D = OP 30/60
 1: Software version: G = ???
-2: Oil pressure calibration number: OIAD = 14 8 0 3 = ???
-3: A2D gain: NNAC = ???
+2: Oil pressure calibration number: OIAD = 14 8 0 3 = 256*3 + 16*14 +8 = 1000 = 1.000
+3: A2D gain: NNAC = 13 13 0 3 =  256*03 + 16*13 +13= 989= 0.989 
 4: Sender Type: AE = 4
 5: Backlight level: AC = 2
-6: Units of measure: AA = psi
+6: Units of measure: AA = psi, otherwise bar
 7: Built in alarms armed: AB = 1 = true
-8: Low pressure alarm value: BLAA = 1 11 0 0 = ???
-9: High pressure alarm value: EBAA =  4 1 0 0 = ??? 
+8: Low pressure alarm value: BLAA = 1 11 0 0 = 256*0 + 16*1 +11= 27 psi
+9: High pressure alarm value: EBAA =  4 1 0 0 = 256*0 + 16*4 +1= 65 psi 
 10: Checksum for Non-Volatile Memory: CM = 2 12 = ???
 11: Oil Pressure: CFAA = 37
-12: Sensor Volts: EPAI = 4 15 0 8 = ???
+12: Sensor Volts: EPAI = 4 15 0 8 = 256*8 + 16*4 +15= 2116=2.116 VDC
 13: Checksum: KI = 10 8
 14: Checksum: 37
 
 Where A=0, B=1, C=2, ... , O=14, P=15
 
 Decode Oil Pressure as:
-CFAA = 16*M + L + 4096*A + 256*A
+CFAA = 16*C + F + 4096*A + 256*A
 CFAA = 16*2 + 5 + 4096*0 + 256*0
 CFAA = 37 psi
 
-//ALPHA is an array map for converting sentence codes from letters to numbers
+//alpha is an array map for converting sentence codes from letters to numbers
 //A=0, B=1, etc
 
+
+*/
+  D: {
+    path: 'propulsion.0.oilPressure',
+    meta: {
+      description: 'CruzPro OP30/OP60 Engine Oil Pressure Gauge',
+      units: 'pa',
+      displayName: 'Engine Oil Pressure',
+      shortName: 'EOP',
+      warnMethod: ['visual'],
+      alarmMethod: ['sound'],
+      gaugeAlarmOn: false,
+      backlight: 0,
+      zones: [],
+      originalValue: null,
+    }
+
+  }, 
+/*
+
+T30/T60 digital temperature gauge sample NMEA sentence:
+
+Sentence: $PBVE,EDOIADOKACABABAAAACAPPCMABCGADABDOAEGL*20
+
+Parse as:
+      0 1   2    3   4  5  6  7   8    9  10  11   12  13 14
+      | |   |    |   |  |  |  |   |    |  |   |    |   |  |    
+$PBVE,x,x,xxxx,xxxx,xx,xx,xx,xx,xxxx,xxxx,xx,xxxx,xxxx,xx*xx
+$PBVE,E D OIAD OKAC AB AB AA AA CAPP CMAB CG ADAB DOAE GL 20
+
+0: Product Code =  E = T30/T60
+1: Software Version # 
+2: Temperature Calibration Number 
+3: A2D gain 
+4: Sender Type
+5: Backlight Level
+6: Units of Measure (0= deg F, non-0= deg C)
+7: Built-in Alarms Armed (AA=0 = NO)
+8: Low temperature alarm value
+9: High temperature alarm value
+10: Checksum for Non-Volatile Memory
+11: Engine Temperature
+12: Sensor Volts
+13: NMEA sentence checksum
+14: Checksum
+
+Where A=0, B=1, C=2, ... , O=14, P=15
+
+Decode Temperature as:
+ADAB = 16*A + D + 4096*A + 256*B
+ADAB = 16*0 + 3 + 4096*0 +256*1
+ADAB = 0 + 3 + 0 + 256
+ADAB = 259 deg F
+
 */
-const ALPHA = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'.split('')
+  E: {
+    path: 'propulsion.0.coolantTemperature',
+    meta: {
```

Comment:
I don't think this will accomplish what you are after here. Check the full that results from this - does it have the meta in place that should be there? 

We don't have a way to update meta information with a delta. @sbender9 was working on this at https://github.com/SignalK/signalk-server-node/pull/481 but that stalled. This use case could be the time to resurrect that line of work.